### PR TITLE
[WebUI] Update Japanese translation files.

### PR DESCRIPTION
**Pull request by kz0817**
This patch add or fixes tranlations into Japanese.

### CODE REVIEW

File: client/conf/locale/ja/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -6,7 +6,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2016-05-27 01:42-0500\n"
+"POT-Creation-Date: 2016-12-20 00:20-0600\n"
 "PO-Revision-Date: 2015-10-30 17:36+0900\n"
 "Last-Translator: Hiroshi Hatake <hatake@clear-code.com>\n"
```
Comment (cosmo0920) [inline client/conf/locale/ja/LC_MESSAGES/django.po]: Could you update this line with your name and e-mail address?

File: client/conf/locale/ja/LC_MESSAGES/djangojs.po [hunk 2]
```diff
@@ -6,7 +6,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2016-05-27 01:42-0500\n"
+"POT-Creation-Date: 2016-12-20 00:20-0600\n"
 "PO-Revision-Date: 2016-01-20 11:37+0900\n"
```
Comment (cosmo0920) [inline client/conf/locale/ja/LC_MESSAGES/djangojs.po]: ditto.

**Comment by cosmo0920** [comment]
Otherwise, it looks good to me.

File: client/conf/locale/ja/LC_MESSAGES/djangojs.po [hunk 3]
```diff
@@ -6,7 +6,7 @@ msgid ""
...
 "Last-Translator: Hiroshi Hatake <hatake@clear-code.com>\n"
```
Comment (cosmo0920) [inline client/conf/locale/ja/LC_MESSAGES/djangojs.po]: ditto.

**Comment by cosmo0920** [comment]
Otherwise, :+1:.

**Comment by kz0817** [comment]
I did.

**Comment by cosmo0920** [comment]
@masa0612 @kz0817 This PR contains a bug which causes build error: https://github.com/project-hatohol/hatohol/pull/2456#discussion_r93374414
Could you follow up this PR?

**Comment by kz0817** [comment]
修正を#2460 でPRした。
ただ、意味のない.po内の行番号の変更も結構混じっているので、少し見難い

**Pull request merged**